week-3/extended-project/nowhere/serve.rb

```diff
@@ -0,0 +1,58 @@
+require 'socket'
+require "erb"
+require_relative 'routes.rb'
+require_relative 'members.rb'
+
+port = 3000
+server = TCPServer.new port
+
+puts "Running server 🚀  in http://localhost:#{port}"
+
+loop do
+  session = server.accept
+  request = []
+  while (line = session.gets) && (line.chomp.length > 0)
+    request << line.chomp
+  end
+
+  puts request[0]
+
+  http_method, path, protocol = request[0].split(' ')
+
+  puts path
+
+  content = 'text/html'
+  status = "200 OK"
+
+  if PAGES.keys.include? path
+    template = File.read('views/' + PAGES[path])
+  elsif path.include? 'member'
```

Comment:
The validation is good, but you could consider change the value to validate since only the server should return the page of each member when pass parameter to the side and the rule to pass parameter in url is that it begin with "?".